Wrapping/Generators/SwigInterface/CMakeLists.txt

```diff
@@ -363,28 +373,46 @@ macro(itk_end_wrap_module_swig_interface)
         )
         set(snake_case_config_file
             "${ITK_WRAP_PYTHON_SNAKE_CONFIG_DIR}/${WRAPPER_LIBRARY_NAME}_snake_case.py")
+        add_custom_command(
+        OUTPUT ${i_files} ${typedef_files} ${idx_files} ${snake_case_config_file} ${ITK_STUB_PYI_FILES}
+        COMMAND ${Python3_EXECUTABLE} ${IGENERATOR}
+          ${mdx_opts}
+          ${swig_libs}
+          -w1 -w3 -w51 -w52 -w53 -w54
+          -A protected -A private
+          -p ${PYGCCXML_DIR}
+          -g ${CASTXML_EXECUTABLE}
+          --snake-case-file "${snake_case_config_file}"
+          --interface-output-dir "${WRAPPER_MASTER_INDEX_OUTPUT_DIR}"
+          --library-output-dir "${WRAPPER_LIBRARY_OUTPUT_DIR}"
+          --submodule-order "${WRAPPER_SUBMODULE_ORDER}"
+          --pyi_dir "${ITK_STUB_DIR}"
+          DEPENDS ${ITK_WRAP_DOC_DOCSTRING_FILES} ${xml_files} ${IGENERATOR} ${typedef_in_files}
+          WORKING_DIRECTORY / # Arguments to WORKING_DIRECTORY may use generator expressions
+        VERBATIM
+      )
       else()
+        unset(ITK_STUB_DIR)
+        unset(ITK_STUB_PYI_FILE)
         unset(snake_case_config_file)
+        add_custom_command(
+          OUTPUT ${i_files} ${typedef_files} ${idx_files}
+          COMMAND ${Python3_EXECUTABLE} ${IGENERATOR}
+            ${mdx_opts}
+            ${swig_libs}
+            -w1 -w3 -w51 -w52 -w53 -w54
+            -A protected -A private
+            -p ${PYGCCXML_DIR}
+            -g ${CASTXML_EXECUTABLE}
+            --interface-output-dir "${WRAPPER_MASTER_INDEX_OUTPUT_DIR}"
+            --library-output-dir "${WRAPPER_LIBRARY_OUTPUT_DIR}"
+            --submodule-order "${WRAPPER_SUBMODULE_ORDER}"
+            DEPENDS ${ITK_WRAP_DOC_DOCSTRING_FILES} ${xml_files} ${IGENERATOR} ${typedef_in_files}
+            WORKING_DIRECTORY / # Arguments to WORKING_DIRECTORY may use generator expressions
+          VERBATIM
+        )
       endif()
 
-      add_custom_command(
-      OUTPUT ${i_files} ${typedef_files} ${idx_files} ${snake_case_config_file} ${ITK_STUB_PYI_FILES}
-      COMMAND ${Python3_EXECUTABLE} ${IGENERATOR}
-        ${mdx_opts}
-        ${swig_libs}
-        -w1 -w3 -w51 -w52 -w53 -w54
-        -A protected -A private
-        -p ${PYGCCXML_DIR}
-        -g ${CASTXML_EXECUTABLE}
-        --snake-case-file "${snake_case_config_file}"
-        --interface-output-dir "${WRAPPER_MASTER_INDEX_OUTPUT_DIR}"
-        --library-output-dir "${WRAPPER_LIBRARY_OUTPUT_DIR}"
-        --submodule-order "${WRAPPER_SUBMODULE_ORDER}"
-        --pyi_dir "${ITK_STUB_DIR}"
-        DEPENDS ${ITK_WRAP_DOC_DOCSTRING_FILES} ${xml_files} ${IGENERATOR} ${typedef_in_files}
-        WORKING_DIRECTORY / # Arguments to WORKING_DIRECTORY may use generator expressions
```

Comment:
`/` has been in use for a while.

Reply:
Yes, this was introduced last year:
https://github.com/hjmjohnson/ITK/commit/12e8d93a1c37cc26cd50d69a31ba3c1b6b82ae96#diff-257f8c554e48118b1a537183ee05207cc10b1d5deb6902cc66814d00e533a983R378
But I don't think it is what created those `M:\RelWithDebInfo`, `M:\Debug` etc folders.

Reply:
Change made as requested.